test/e2e/basic_workflow_test.go

```diff
@@ -23,109 +23,110 @@ import (
 
 	"gotest.tools/assert"
 
+	"knative.dev/client/lib/test/integration"
 	"knative.dev/client/pkg/util"
 )
 
 func TestBasicWorkflow(t *testing.T) {
 	t.Parallel()
-	test, err := NewE2eTest()
+	it, err := integration.NewIntegrationTest()
 	assert.NilError(t, err)
 	defer func() {
-		assert.NilError(t, test.Teardown())
+		assert.NilError(t, it.Teardown())
 	}()
 
-	r := NewKnRunResultCollector(t)
+	r := integration.NewKnRunResultCollector(t)
```

Comment:
I just wonder if KnRunResultCollector could be move to KnIntegrationtest `Test` as a field, because every `Test` shall have a `KnRunResultCollector`. If `KnRunResultCollector` is associated with `Test` and `KnRunResultCollector.AssertError(out)` can be moved to `integration.go` as `Test.AssertError(out)`, we don't need to send two parameters to every test method, e.g.
```
func serviceListEmpty(t *testing.T, it *integration.Test, r *integration.KnRunResultCollector)
func serviceCreate(t *testing.T, it *integration.Test, r *integration.KnRunResultCollector, serviceName string)
```
After all, we have so many such kind of test methods. Saving 1 parameter could save a big.